From: Christoph Anton Mitterer <mail@christoph.anton.mitterer.name>
Date: Wed, 30 Jun 2021 13:34:39 +0200
Subject: meson: minor cleanup

The file itself is already only included when HAVE_MICROHTTPD is set.

Signed-off-by: Christoph Anton Mitterer <mail@christoph.anton.mitterer.name>
(cherry picked from commit a2142fa60d6bf2408dd42bce637e3ae84b7dad76)
---
 sysusers.d/systemd-remote.conf    |  8 ++++++++
 sysusers.d/systemd-remote.conf.in | 10 ----------
 2 files changed, 8 insertions(+), 10 deletions(-)
 create mode 100644 sysusers.d/systemd-remote.conf
 delete mode 100644 sysusers.d/systemd-remote.conf.in

diff --git a/sysusers.d/systemd-remote.conf b/sysusers.d/systemd-remote.conf
new file mode 100644
index 0000000..ca20c24
--- /dev/null
+++ b/sysusers.d/systemd-remote.conf
@@ -0,0 +1,8 @@
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+u systemd-journal-remote  - "systemd Journal Remote"
diff --git a/sysusers.d/systemd-remote.conf.in b/sysusers.d/systemd-remote.conf.in
deleted file mode 100644
index dae1077..0000000
--- a/sysusers.d/systemd-remote.conf.in
+++ /dev/null
@@ -1,10 +0,0 @@
-#  This file is part of systemd.
-#
-#  systemd is free software; you can redistribute it and/or modify it
-#  under the terms of the GNU Lesser General Public License as published by
-#  the Free Software Foundation; either version 2.1 of the License, or
-#  (at your option) any later version.
-
-{% if HAVE_MICROHTTPD %}
-u systemd-journal-remote  - "systemd Journal Remote"
-{% endif %}
